pkgs/applications/audio/ardour/6.nix

```diff
@@ -0,0 +1,190 @@
+{ lib, stdenv
+, fetchgit
+, alsa-lib
+, aubio
+, boost
+, cairomm
+, cppunit
+, curl
+, dbus
+, doxygen
+, ffmpeg
+, fftw
+, fftwSinglePrec
+, flac
+, glibc
+, glibmm
+, graphviz
+, gtkmm2
+, harvid
+, itstool
+, libarchive
+, libjack2
+, liblo
+, libogg
+, libpulseaudio
+, librdf_raptor
+, librdf_rasqal
+, libsamplerate
+, libsigcxx
+, libsndfile
+, libusb1
+, libuv
+, libwebsockets
+, libxml2
+, libxslt
+, lilv
+, lrdf
+, lv2
+, makeWrapper
+, pango
+, perl
+, pkg-config
+, python3
+, readline
+, rubberband
+, serd
+, sord
+, soundtouch
+, sratom
+, suil
+, taglib
+, vamp-plugin-sdk
+, wafHook
+, xjadeo
+, videoSupport ? true
+}:
+stdenv.mkDerivation rec {
+  pname = "ardour";
+  version = "6.9";
+
+  # don't fetch releases from the GitHub mirror, they are broken
+  src = fetchgit {
+    url = "git://git.ardour.org/ardour/ardour.git";
+    rev = version;
+    sha256 = "0vlcbd70y0an881zv87kc3akmaiz4w7whsy3yaiiqqjww35jg1mm";
+  };
```

Comment:
The fetchFromGitea wrapper should be used here instead.

Reply:
Apparently the ardour repo block downloading github-style release tarballs: https://tracker.ardour.org/view.php?id=7328

which the `fetchFromGitea` wrapper relies on:

https://github.com/NixOS/nixpkgs/blob/915fd63594ffc701f6f30b3af4767338fc666e61/pkgs/build-support/fetchgitea/default.nix#L1-L7

https://github.com/NixOS/nixpkgs/blob/915fd63594ffc701f6f30b3af4767338fc666e61/pkgs/build-support/fetchgithub/default.nix#L53

Thus using `fetchFromGitea` results in an empty archive. I'll add a comment above the `fetchgit` call explaining this.

Reply:
Done in https://github.com/NixOS/nixpkgs/pull/196290/commits/db616b233e38e5456ecae20ceaf4008ca8aadceb.